testing/ossp-uuid/APKBUILD

```diff
@@ -0,0 +1,32 @@
+# Contributor: "Andrzej Trzaska <atrzaska2@gmail.com>"
+# Maintainer: "Andrzej Trzaska <atrzaska2@gmail.com>"
+pkgname=ossp-uuid
+pkgver=1.6.2
+pkgrel=0
+pkgdesc="Open Source Software Project Universally Unique Identifier (UUID)"
+url="http://www.ossp.org/pkg/lib/uuid/"
+arch="all"
+license="GPL-2.0"
+depends=""
+install=""
```

Comment:
no reason for empty definitions

Reply:
removed empty `depends` and `install` definitions